src/EditorFeatures/CSharpTest/Diagnostics/GenerateVariable/GenerateVariableTests.cs

```diff
@@ -7154,5 +7154,27 @@ public Foo(String foo)
     public string String { get; private set; }
 }");
         }
+
+        [WorkItem(18275, "https://github.com/dotnet/roslyn/issues/18275")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddImport)]
+        public async Task TestContextualKeyword1()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"
+namespace N
+{
+    class nameof
```

Comment:
:bulb: You could make this a `Theory` instead of `Fact`, and provide inline data for each keyword. You can see an example of this in the diff for #18424. The same applies to other cases where `nameof` is hard-coded for the test.